index.html

```diff
@@ -0,0 +1,249 @@
+                        </h1>
+                    </div>
+                    <div class="card__body">
+                        the quick fox jumps over the lazy 
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card__header">
+                        <img src="./assets/svgs/users.svg" alt="Bag">
+                        <h1 class="card__title">
+                            Digital Marketing
+                        </h1>
+                    </div>
+                    <div class="card__body">
+                        the quick fox jumps over the lazy 
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card__header">
+                        <img src="./assets/svgs/chart.svg" alt="Bag">
+                        <h1 class="card__title">
+                            National top 50 firms
+                        </h1>
+                    </div>
+                    <div class="card__body">
+                        the quick fox jumps over the lazy 
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card__header">
+                        <img src="./assets/svgs/award.svg" alt="Bag">
+                        <h1 class="card__title">
+                            Web Development
+                        </h1>
+                    </div>
+                    <div class="card__body">
+                        the quick fox jumps over the lazy 
+                        dog the quick fox jumps over the 
+                        lazy dog
+                    </div>
+                </div>
+            </div>
+        </div>
+    
+        <div class="video">
+            <div class="video__headline">
+                <h1 class="site-header site-header--medium site-header--center" >
+                    We are providing best 
+                    <br>
+                    business service.
+                </h1>
+                <div class="site-content">
+                    Problems trying to resolve the conflict between <br>
+                    the two major realms of Classical physics: Newtonian mechanics 
+                </div>
+            </div>
+            <div class="main-video">
+                <div class="video__image">
+                    <img class="site-video" src="./assets/svgs/video.svg">
+                </div>
+            </div>
+            <div class="container">
+                <h1 class="site-header site-header--medium site-header--center" >
+                    What Clients Say
+                </h1>
+                <div class="site-content">
```

Comment:
how about you name this depending on container, one would think they are very separated while it is in the container

Reply:
Am going to hide this all comment because my pr is still in progress